Test/RemotePcapTests.cs

```diff
@@ -107,25 +106,6 @@ public void PwdAuthTest()
 
     }
 
-    class PlainCredential : NetworkCredential
-    {
-        public PlainCredential(string username, string password, string domain)
-           : base(username, password, domain)
-        {
-
-        }
-
-        public override string ToString()
```

Comment:
`PlainCredential` is still needed because of the `ToString()` override, otherwise test runner will report wrong test count, and may skip running some tests.

Either move the `ToString()` override to `RemoteAuthentication` or restore PlainCredential` and make it inherit from `RemoteAuthentication`

Reply:
Why does the test runner need this override? If we do move it into RemoteAuthentication what do we use for the domain as RemoteAuthentication has username, password, and type.

Reply:
@chmorgan because it is used as a `ValueSource` for `PcapInterfaceNullAuthTest`, the difference is most visible in Visual Studio Test Explorer:
master
![image](https://user-images.githubusercontent.com/1814900/104838428-2b172d80-58bb-11eb-98d2-ae93203205ff.png)
current branch: only two test cases show up instead of 3
![image](https://user-images.githubusercontent.com/1814900/104838448-4a15bf80-58bb-11eb-83f1-fb16951bbdc4.png)

Reply:
VS Mac doesn't show it the same way but I see 3:
![image](https://user-images.githubusercontent.com/1211904/104844707-c69aa500-589f-11eb-895f-870d0f701c48.png)

Reply:
You could (as another solution) replace `ValueSource` with a `foreach` and have the test case non parametrized.

Reply:
Does it look like VS Mac is doing the right thing here though? The parameters match the ones you have.

Reply:
Ironically, VS Mac is doing better than VS Windows, however due to the test cases having the exact same name, it's not possible to tell what values each test case is running with.

Reply:
Alright, let me put back PlainCredential and see how it looks here, give me a few min.

Reply:
@kayoub5 alright, now I see what the purpose was there. I documented the 'why' for the class too. How does that look?